scripts/release-post.js

```diff
@@ -181,12 +181,21 @@ function renderPost (results) {
 function writeToFile (results) {
   const filepath = path.resolve(__dirname, '..', 'locale', 'en', 'blog', 'release', `v${results.version}.md`)
 
-  if (fs.existsSync(filepath)) {
-    return Promise.reject(new Error(`Release post for ${results.version} already exists!`))
-  }
+  return new Promise(function (resolve, reject) {
+    fs.access(filepath, fs.R_OK | fs.W_OK, function (err) {
+      if (!err) {
```

Comment:
@lpinca If there is no error, it means that the file exists. [`fs.access`](https://nodejs.org/api/fs.html#fs_fs_access_path_mode_callback)